omniport/core/session_auth/models/session_map.py

```diff
@@ -76,7 +76,7 @@ def create_session_map(request, user, new=True):
         session_map.location = location
 
         # Get the browser, operating system and device from the user-agent
-        user_agent_string = request.META['HTTP_USER_AGENT']
+        user_agent_string = request.META.get('HTTP_USER_AGENT','Unknown')
```

Comment:
```suggestion
        user_agent_string = request.META.get('HTTP_USER_AGENT', 'Unknown')
```

Reply:
Also, I'm not sure `get_agent_information` will handle `'Unknown'` passed to it.

Reply:
The [function](https://github.com/IMGIITRoorkee/omniport-backend/blob/727bdb2c14a8fe8eded14a23e6156e712e078bfc/omniport/core/session_auth/utils/user_agent.py#L6) handles the case `Unknown` passed to it. I tried running it in the shell also, with the following response. 
![image](https://user-images.githubusercontent.com/34851451/81635060-dd921900-942d-11ea-94a3-fd2d09640b97.png)